components/sup/src/manager/service/spec.rs

```diff
@@ -417,7 +417,7 @@ impl Default for ServiceSpec {
             group: DEFAULT_GROUP.to_string(),
             application_environment: None,
             bldr_url: DEFAULT_BLDR_URL.to_string(),
-            channel: STABLE_CHANNEL.to_string(),
+            channel: ChannelIdent::stable().to_string(),
```

Comment:
`String` -> `ChannelIdent`

Reply:
Also, just look at all those other default strings just begging to be refactored to types 😄

Reply:
[Done](https://github.com/habitat-sh/habitat/pull/6074/commits/cbebb204f2ee3b0b2cafe139cbf99ed1baf73d94#diff-67161b6446fbddc099ba24f430c21d4bR420)